### PR TITLE
Refactor style and adding commit

### PR DESCRIPTION
**Pull request by louisgv**

### CODE REVIEW

File: goon-gate.web/src/components/ChannelContainer/index.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+
+import styled from 'styled-components';
+
+// TODO: Extract these out as reusable component if necessary
```
Comment (ColdSauce) [inline goon-gate.web/src/components/ChannelContainer/index.jsx]: Could you create a github issue for this and link it in here.

File: goon-gate.web/src/pages/ImageViewer/index.jsx [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
+`
+
+export default class ImageViewer extends PureComponent {
 	render() {
 		return (
-			<div className="ImageViewer">
-				<IPFSImageWrapper />
-			</div>
+			<Container>
+				  <IPFSImageWrapper />
```
Comment (ColdSauce) [inline goon-gate.web/src/pages/ImageViewer/index.jsx]: Remove trailing whitespace

File: goon-gate.web/src/sections/ImageUploader/index.css [hunk 3]
```diff
@@ -5,6 +5,9 @@
   align-items: center;
 }
 .uploadImage input {
+  text-align: center;
+  margin: auto;
+
```
Comment (ColdSauce) [inline goon-gate.web/src/sections/ImageUploader/index.css]: remove line

File: goon-gate.web/src/sections/Messenger/index.jsx [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+import ChannelContainer from 'components/ChannelContainer';
+
+import ChannelCell from 'components/ChannelCell'
+import MessageCell from 'components/MessageCell'
+
+const CHANNELS = new Array(7)
+	.fill(0)
+	.map((item, i) => `#Channel ${i}`);
+
+// replace with store or some data api
```
Comment (ColdSauce) [inline goon-gate.web/src/sections/Messenger/index.jsx]: can you add a issue number to this

Comment (louisgv) [reply goon-gate.web/src/sections/Messenger/index.jsx]: Done

Comment (louisgv) [reply goon-gate.web/src/sections/Messenger/index.jsx]: Done

**Pull request merged**